dune-project

```diff
@@ -10,7 +10,6 @@
  (synopsis "Deploy other OCurrent pipelines and services")
  (depends
   (ocaml (>= 4.14))
-  ; Provided by git submodules
   (current (>= 0.6.4))
```

Comment:
Should we bump up the version to the latest one, since we are moving from a submodule to a published version?